CMakeLists.txt

```diff
@@ -346,6 +346,10 @@ if (ENABLE_IWYU)
   endif()
 endif()
 
+add_custom_target(reformat
+    COMMAND Scripts/reformat.sh ${CMAKE_SOURCE_DIR}
```

Comment:
This needs to be changed to
```diff
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6f5e9081a..49c1a2972 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -347,7 +347,7 @@ if (ENABLE_IWYU)
 endif()

 add_custom_target(reformat
-    COMMAND Scripts/reformat.sh ${CMAKE_SOURCE_DIR}
+  COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/Scripts/reformat.sh ${CMAKE_SOURCE_DIR}
 )

 add_compile_options(-Wall)
```
Otherwise when executing from a build folder it doesn't find the script

Reply:
Thanks.